src/commands/server/delete.ts

```diff
@@ -97,7 +98,9 @@ export default class Delete extends Command {
   }
 
   async isDeletionConfirmed(flags: any): Promise<boolean> {
-    const cluster = KubeHelper.KUBE_CONFIG.getCurrentCluster()
+    const kc = KubeHelper.initializeKubeConfig()
```

Comment:
Sorry, why do you need initialize kube config? I think we can get cluster directly  kc.kubeConfig.getCurrentCluster() ?

Reply:
``` javascript
const kc = new KubeHelper(flags)
const cluster = kc.kubeConfig.getCurrentCluster()
```

Reply:
Is better